example02/reducers/reducer.js

```diff
@@ -0,0 +1,34 @@
+const init = {
+    boxes: []
+};
+
+export default function reducer(state = init, action, args) {
+    const random = Math.random;
+    const floor = Math.floor;
```

Comment:
I'm not a big fan of shortcuts like this. I think they ultimately hurt readability because `Math.random` reads like an idiom whereas `random` could be anything :) I won't block on this, though.

Reply:
I do it as an optimization. It's acting as a direct reference to the method vs having to go through two separate references before calling the function. But yeah, you make a fair point. I'll leave them for now, but if users complain then I will be obliged to remove them.